code/modules/martial_arts/sleeping_carp.dm

```diff
@@ -59,6 +60,7 @@
 		playsound(get_turf(D), 'sound/weapons/punch1.ogg', 50, 1, -1)
 		if(prob(80))
 			A.say(pick("SURRPRIZU!","BACK STRIKE!","WOPAH!", "WATAAH", "ZOTA!", "Never turn your back to the enemy!"))
+			add_attack_logs(A, D, "Melee attacked with martial-art [src] :  Back Kick", ATKLOG_ALL)
```

Comment:
Should be indented outside the if scope. Now it depends on if the user is forced to speak some random words. Same for the others below

Reply:
I'll fix it as soon as I can, thanks for pointing it out.